### PR TITLE
Make the replications persistent

### PR DESCRIPTION
**Pull request by Gara64**
Instead of using the _replicate API, use the _replicator database, which allows to make the replications process persistent, even after a server reboot.

### CODE REVIEW

File: server/helpers/db_connect_helper.coffee [hunk 1]
```diff
@@ -32,11 +32,20 @@ setup_credentials = ->
...
-        credentials = setup_credentials()
+        credentials = setup_credentials 'cozy'
         connection = new cradle.Connection credentials
         db = connection.database credentials.db
 
     return db
+
+exports.db_replicator_connect = ->
+    if not replicator?
+        credentials = setup_credentials '_replicator'
```
Comment (frankrousseau) [inline server/helpers/db_connect_helper.coffee]: Could use camel cases instead of snake cases to name functions?

Comment (Gara64) [inline server/helpers/db_connect_helper.coffee]: Sure, I used snake case here to be consistant with the existing function `db_connec`t, but I can change to camel case if you prefer.

Comment (frankrousseau) [inline server/helpers/db_connect_helper.coffee]: Yep I prefer. I think it was a mistake too with the `db_connect` method.

Comment (Gara64) [inline server/helpers/db_connect_helper.coffee]: Ok, done

Comment (frankrousseau) [inline server/helpers/db_connect_helper.coffee]: I was thinking about the `setup_credentials` function.

Comment (Gara64) [inline server/helpers/db_connect_helper.coffee]: My bad, I changed

**Comment by frankrousseau** [comment]
Thx @Gara64 !

**Pull request merged**